### DISCUSSION TITLE
is it possible to has list's as column in view collection

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#4038] [Q&A] [answered]
example:
```
base colelction records:
user_id : 1, team_id : 1
user_id : 2, team_id : 1
user_id:3, team_id : 2
```

view_collections records should be
```
team_id: 1, user_id: [1,2]
team_id:2 , user_id [3]
```

**Comment by ganigeorgiev** [comment, marked as answer]
You can use a combination of [`json_group_array(user_id)`](https://www.sqlite.org/json1.html#jgrouparray) and `GROUP BY team_id`. Something like (_I haven't tested it_):

```sql
SELECT
  team_id as id,
  team_id,
  json_group_array(user_id) as user_ids
FROM yourCollection
GROUP BY team_id
```

Note that the `json_group_array` expression will be treated as plain `json` field, meaning that `expand` and other relation related filter utils will not work for the `user_ids`.
